webpages/settings/index.js

```diff
@@ -256,17 +256,10 @@ window.addEventListener("keydown", function (e) {
     document.querySelector("#searchBox").focus();
   }
 });
-theme = chrome.storage.sync.get(["globalTheme"], function (r) {
-  console.log(theme);
-});
-console.log(theme);
-while ((this.selectedMode == "dark") & (theme !== false)) {
-  chrome.storage.sync.set({ globalTheme: false }, function () {
-    console.log(this.selectedMode + " mode set");
-  });
-}
-while ((this.selectedMode == "light") & (theme !== true)) {
-  chrome.storage.sync.set({ globalTheme: true }, function () {
-    console.log(this.selectedMode + " mode set");
-  });
-}
+
+chrome.storage.sync.get({ globalTheme }, function (theme) {
+    while (theme.globalTheme !== this.selectedMode)
```

Comment:
While??

Reply:
Totally changed in new commit. I went with a simple button. If you want that changed, do it yourself. 😄